EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -126,13 +126,14 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(difference, [0, 5, 6, 7, 8])
     }
 
+    /// WARNING: [SE-0121] Remove Optional Comparison Operators
     func testOptionalEquatable() {
         let a: [Int]? = [1, 2, 3]
```

Comment:
@goktugyil yes, you are right, but `[T]? == [T]?` which is defined in ArrayExtensions are not visible and working even if it's public, it crashes when I run tests